src/widget/wtracktableview.cpp

```diff
@@ -565,6 +570,14 @@ void WTrackTableView::mouseMoveEvent(QMouseEvent* pEvent) {
     if (!trackModel) {
         return;
     }
+
+    const auto delta = m_mousePressPosition - pEvent->position();
+    const qreal threshold = 5;
+    if (delta.x() * delta.x() + delta.y() * delta.y() < threshold * threshold) {
```

Comment:
It took me a moment to understand this formula. Can you add comment about  the used "Pythagorean Theorem"

Reply:
I added an std::sqrt and an explicitly named variabled to make things more obvious.